android/app/build.gradle

```diff
@@ -108,4 +115,30 @@ flutter {
 }
 
 dependencies {
-}
+    /* FCM */
+    implementation 'com.google.firebase:firebase-messaging-ktx:23.3.1'
+
+    /* WorkManager */
+    def work_version = "2.9.0"
+
+    // (Java only)
+    implementation "androidx.work:work-runtime:$work_version"
+
+    // Kotlin + coroutines
+    implementation "androidx.work:work-runtime-ktx:$work_version"
+
+    // optional - RxJava2 support
+    implementation "androidx.work:work-rxjava2:$work_version"
+
+    // optional - GCMNetworkManager support
+    implementation "androidx.work:work-gcm:$work_version"
+
+    // optional - Test helpers
+    androidTestImplementation "androidx.work:work-testing:$work_version"
+
+    // optional - Multiprocess support
+    implementation "androidx.work:work-multiprocess:$work_version"
```

Comment:
Erdem could you elaborate why we need all these optional implementatinos?

Reply:
We don't need all of them. They were declared on the official documentation and unused dependencies will be removed later on.

Reply:
Done [Remove unnecessary dependencies](https://github.com/breez/c-breez/pull/737/commits/e8ee75199ca39fb5f38d6d0a2b5529162a8bbf28)